distribution/bin/to_bytes.gawk

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use gawk because gnu awk can't extract regexp groups; gawk has `match`
```

Comment:
@huanwei Is there any script to test whether `gawk` exists and install it if missed?
My concern is that our users may use this script on their OS instead of Docker container, so we may need to solve the dependencies of `gawk`. So I'm wondering whether we can calculate values of available memory without `gawk` by using bash built-in functions and commands.

And there is another issue, the Apache RocketMQ has `cmd` script for Windows users, I think we should make scripts of Unix/Linux and Windows working in same logic.

Reply:
ping @huanwei @vongosling

Reply:
@XiaoZYang Thanks for your suggestion. For Unix/Linux I have changed the scripts to use the `awk`.

Reply:
@huanwei cool, would you mind change your pr merge target to develop branch. It has a little difference with external projects.

Reply:
@huanwei Oh, I'd like to have a try on Windows tomorrow 👍